eip_operator_shared/Cargo.toml

```diff
@@ -28,6 +28,5 @@ tracing = "0.1"
 tracing-opentelemetry = "0.19"
 tracing-subscriber = { version = "0.3", features = ["registry", "env-filter", "json"] }
 
-rtnetlink = { git = "https://github.com/MaterializeInc/netlink.git", branch = "priority_support" }
```

Comment:
Originally I. had removed this just because I couldn't get it to compile on my mac. I then realized it was brought in for a seemingly unused error message.

Reply:
Ok this is needed for the cilium eip no masquerade stuff. I'll bring it back and push.